RELEASING.md

```diff
@@ -0,0 +1,72 @@
+How To Release
+==============
+
+Due to Maven Central's very particular requirements, the release process is a bit
+elaborate and requires a good deal of local configuration.  This guide should walk
+you through it.  It won't do anyone outside of KeepSafe any good, but the workflow
+is representative of just about any project deploying via Sonatype.
+
+We currently deploy to Maven Central (via Sonatype's OSS Nexus instance).
+
+### Prerequisites
+
+1. A *published* GPG code-signing key
+1. A Sonatype Nexus OSS account with permission to publish in com.getkeepsafe
+1. A plugins.gradle.org account with permission to publish in com.getkeepsafe
```

Comment:
plugins.gradle.org isn't needed for relinker - it's only for publishing Gradle plugins to Gradle's own repo.